oarepo_oidc_einfra/remote.py

```diff
@@ -235,12 +242,23 @@ def autocreate_user(
         "affiliations": "",
         "full_name": account_info["user"]["profile"]["full_name"],
     }
+    user_preferences = {
+        "visibility": "public",
```

Comment:
Do we need everyone public by default? (their e-mails might be searchable by everyone?)